examples/build_examples.sh

```diff
@@ -109,6 +109,7 @@ main() {
     "./examples/container/datarw"
     "./examples/container/hello"
     "./examples/container/memeater"
+    "./examples/container/resource/resource_user"
```

Comment:
I'd rename this to `resource_hello` or `simple` to indicate that this is the most trivial use case.